src/index.css

```diff
@@ -3,6 +3,7 @@
     --mainColor: black;
     --debugHeaderColor: gray;
     --backgroundColor: rgb(245, 245, 245);
+    --errorMessageColor: red;
```

Comment:
can you make these `--kebab-case`? I realize I didn't make my first ones into kebab case but we should probably do that.